Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -1644,6 +1644,16 @@
       )], HAVE_ANDROID_MEDIA_HYBRIS="no")
 ])
 
+HAVE_MIRCLIENT="no"
+if test "x$HAVE_ANDROID_MEDIA_HYBRIS" = "xyes"; then
+    PKG_CHECK_MODULES(MIRCLIENT, mirclient,
+                      HAVE_MIRCLIENT="yes",
+                      AC_MSG_ERROR([Android Media Hybris requested but mirclient not found]))
+fi
+
+AC_SUBST(MIRCLIENT_CFLAGS)
+AC_SUBST(MIRCLIENT_LIBS)
+
 dnl *** AppleMedia (OS X and iOS) ***
 translit(dnm, m, l) AM_CONDITIONAL(USE_APPLE_MEDIA, true)
 HAVE_APPLE_MEDIA="no"
Index: b/ext/mir/Makefile.am
===================================================================
--- a/ext/mir/Makefile.am
+++ b/ext/mir/Makefile.am
@@ -3,6 +3,7 @@
 libgstmirsink_la_SOURCES = gstmirsink.c mirpool.c
 libgstmirsink_la_CFLAGS = $(GST_CFLAGS) $(GST_PLUGINS_BASE_CFLAGS) \
                                $(UBUNTU_PLATFORM_CFLAGS) \
+                               $(MIRCLIENT_CFLAGS) \
                                $(MIR_CFLAGS) \
                                -I../../gst-libs/
 libgstmirsink_la_LIBADD = $(GST_PLUGINS_BASE_LIBS) \
@@ -10,6 +11,7 @@
                                $(top_builddir)/gst-libs/gst/mir/libgstmiralloc-$(GST_API_VERSION).la \
                                -lmedia \
                                $(EGL_LIBS) $(EGLGLES_LIBS) \
+                               $(MIRCLIENT_LIBS) \
                                $(MIR_LIBS)
 libgstmirsink_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
 libgstmirsink_la_LIBTOOLFLAGS = $(GST_PLUGIN_LIBTOOLFLAGS)
Index: b/sys/androidmedia/Makefile.am
===================================================================
--- a/sys/androidmedia/Makefile.am
+++ b/sys/androidmedia/Makefile.am
@@ -27,6 +27,7 @@
 	$(GST_BASE_CFLAGS) \
 	$(GST_CFLAGS) \
 	$(UBUNTU_PLATFORM_CFLAGS) \
+	$(MIRCLIENT_CFLAGS) \
 	$(ORC_CFLAGS)
 libgstandroidmedia_la_LIBADD = \
 	$(GST_PLUGINS_BAD_LIBS) \
@@ -37,6 +38,7 @@
 	-lmedia \
 	$(GST_BASE_LIBS) \
 	$(GST_LIBS) \
+	$(MIRCLIENT_LIBS) \
 	$(ORC_LIBS) \
 	$(top_builddir)/gst-libs/gst/mir/libgstmiralloc-@GST_API_VERSION@.la
 libgstandroidmedia_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS)
